faq/index.md

```diff
@@ -16,37 +16,57 @@ You can always fall back to using email authentication to login. If your device
+Private key references are stored in [indexeddb](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Using_IndexedDB),
+To see all the key references stored select the storage tab in web developer tools.
+
+Where in your computers filesystem the browser stores depends on both the browser and the operating system you are using.
+For example Firefox stores data in a `storage` directory located under `profile`. Full details are explained [here](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API/Browser_storage_limits_and_eviction_criteria#Where_is_the_data_stored)
+
 <a name="can-did-be-whitelabelled-on-my-website"></a>
-### Can DID be whitelabelled on my website?
+### Can DID.app be whitelabelled on my website?
 
-The style and appearance of DID on your website can be customised. Any customers on a [paid plan](https://did.app/#pricing) can also remove the 'Powered by DID' link.
+The style and appearance of DID.app on your website can be customised. Any customers on a [paid plan](https://did.app/#pricing) can also remove the 'Powered by DID' link.
 
 <a name="why-wouldnt-i-just-use-social-logins"></a>
 ### Why wouldn't I just use social logins?
-DID is more convenient to use than social login.  Users that do not have Facebook or Google accounts cannot use those social logins.  If the user does have an account with Facebook or Google they still need to log into it using an email address and password.
+DID.app is more convenient to use than social login.  Users that do not have Facebook or Google accounts cannot use those social logins.  If the user does have an account with Facebook or Google they still need to log into it using an email address and password.
 
 Login method choice can cause confusion for the user.  Some confusion can also occur if a user has previously signed up with Facebook but now tries to login with Google or vice versa.
 
 There is also a security threat to consider.  If a Google or Facebook account is compromised the hacker now has access to all the websites which the social login has been used to grant access.  
 
 Some users are also aware of the privacy concerns around the business models of Facebook and Google and their motivations for providing login functionality for tracking purposes.  
 
-DID offers a simple, secure solution that is easier to use than social logins and respects the user’s privacy.
+DID.app offers a simple, secure solution that is easier to use than social logins and respects the user’s privacy.
+
+### Why does DID.app send a magic link rather than a code?
+
+
+When copying a code it is possible for an attacker to mislead a user in to copying a code into the attackers website.
+This enables the attacker to compromise the users account.
+
+Requiring users to click on a link is more secure than asking them to copy a code.
+The link includes the correct domain and is more resistant to phishing attack described above.
```

Comment:
```suggestion
The link includes the correct domain and is more resistant to kind of phishing attack described above.
```

Reply:
@richardesigns this doesn't quite make sense to me

Reply:
@CrowdHailer You're right, I missed the 'the'.

Reply:
Ah yes, thanks.